drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/ActionWorkItemExecuteColumnSynchronizerTest.java

```diff
@@ -16,21 +16,52 @@
 
 package org.drools.workbench.screens.guided.dtable.client.widget.table.model.synchronizers.impl;
 
+import java.util.HashMap;
 import java.util.List;
 
+import org.appformer.project.datamodel.oracle.DataType;
+import org.appformer.project.datamodel.oracle.FieldAccessorsAndMutators;
+import org.appformer.project.datamodel.oracle.ModelField;
 import org.drools.workbench.models.guided.dtable.shared.model.ActionWorkItemCol52;
 import org.drools.workbench.models.guided.dtable.shared.model.BaseColumnFieldDiff;
 import org.drools.workbench.screens.guided.dtable.client.widget.table.columns.BaseMultipleDOMElementUiColumn;
 import org.drools.workbench.screens.guided.dtable.client.widget.table.columns.BooleanUiColumn;
 import org.drools.workbench.screens.guided.dtable.client.widget.table.model.synchronizers.ModelSynchronizer;
 import org.junit.Test;
+import org.kie.workbench.common.widgets.client.datamodel.AsyncPackageDataModelOracle;
 import org.uberfire.ext.wires.core.grids.client.model.GridColumn;
 import org.uberfire.ext.wires.core.grids.client.model.impl.BaseGridCellValue;
 
 import static org.junit.Assert.*;
 import static org.mockito.Mockito.*;
 
-public class ActionWorkItemColumnSynchronizerTest extends BaseSynchronizerTest {
+public class ActionWorkItemExecuteColumnSynchronizerTest extends BaseSynchronizerTest {
+
+    @Override
+    protected AsyncPackageDataModelOracle getOracle() {
+        final AsyncPackageDataModelOracle oracle = super.getOracle();
+        oracle.addModelFields(new HashMap<String, ModelField[]>() {
+                                  {
+                                      put("Applicant",
+                                          new ModelField[]{
+                                                  new ModelField("this",
```

Comment:
Please assume usage of org.drools.workbench.screens.guided.rule.client.util.ModelFieldUtil.modelField method

Reply:
Also, since the map apparently only contains one key-value pair, `Collections.singletonMap()` will remove a lot of the boilerplate.